### ISSUE TITLE
Bass Flute Musesounds absolutely not in time (Long-standing issue)

### ISSUE THREAD

**Issue by as-Circe** [musescore/MuseScore#23437]
### Issue type

Muse Sounds bug

### Bug description

The Bass Flute Musesounds sound font has been incredibly out of time since I can remember. It used to be the same for the Alto Flute but that was fixed in a previous update. I've noticed it on several scores going back several months, it's a really pervasive issue. I would love to use and recommend Musesounds, because it's one of the only free, high-quality flute soundfonts packs out there. 

For the record, the actual sound is marvellous. It's just really out of time. 

### Steps to reproduce

1. Open a new score
2. Select Bass Flute
3. Write some various note values
4. Turn on Metronome
5. Listen in horror

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/110109337/95f046d4-b959-4240-9f52-873939730d3d

Audio quality on the screen recording is very bad, I know, but it almost sounds like I have swing 16ths on (which I don't) 
It's bad
Please fix it
<3

### MuseScore Version

4.3.2-241630832

### Regression

I don't know

### Operating system

macOS Sonoma 14.2.1

**Comment by Dima-S-Jr** [comment]
Am I the only one who can't view this recording? @as-Circe, is this a corrupted video file?

**Comment by MovableFormula** [comment]
> Am I the only one who can't view this recording? @as-Circe, is this a corrupted video file?
> 
>  01-49-57.mp4

Works for me

**Comment by as-Circe** [comment]
> Am I the only one who can't view this recording? @as-Circe, is this a corrupted video file?

@Dima-S-Jr I don't believe it is, but you should be able to follow the reproduction instructions above to get an idea of the issue.

**Comment by farvardin** [comment]
I have something similar with muse sound and the oboe sound. 
With a fresh new partition and those notes, the A in the red part is not sounding if precessed with an upper E.  It sounds ok if precessed with an upper C. Strange.

![image](https://github.com/musescore/MuseScore/assets/477243/af15d783-671a-4215-9399-915bc27f0ebe)

**Comment by MarcSabatella** [comment]
That’s a separate issue - see https://github.com/musescore/MuseScore/issues/22888

**Comment by as-Circe** [comment]
This is still an issue and has been now for over a year. I love MuseScore Studio but this is frankly awful. This has been fixed for the Alto Flute before, exact same issue of being out of time. Please fix the bass flute soundfont.
